src/activities/components/UserSlot.vue

```diff
@@ -3,18 +3,17 @@
     class="user-slot-wrapper"
     :class="{ greyedOut: !showJoin, active: showJoin }"
     :style="{ width: size + 'px', height: size + 'px' }"
+    @click.stop="showJoin ? $emit('join') : null"
```

Comment:
I now understand moving the click handler up here is to achieve _don't go to activity detail when clicking on an empty slot_

I'm not totally convinced that is an improvement - as it reduces the clickable area to get into the chat, and I wouldn't have an expectation that they grey slots are clickable in a different way to the whole item.

On the other hand, I'm totally happy to go with this if you think it's an improvement. My point is only very minor. So please keep it if you want :)

Reply:
I put it there because I started with the changes from your PR. There you say:
> put the click handler on a higher up element (as the hover element is no longer shown)

I would also prefer to have only a dedicated button "open conversation" and not the whole box clickable. Maybe we can talk about this on Monday...

Reply:
Haha, I forgot what my changes were :)

Yup, would be happy for UX improvements for activity chat opening. It could lead to a bigger change, so maybe to get this one merged we keep it as you've done it here (making it so the empty slots aren't clickable). What do you think?

Reply:
That's what I had in mind too :)